apps/confidential/confidential.go

```diff
@@ -428,3 +428,9 @@ func (cca Client) RemoveAccount(account Account) error {
 	cca.base.RemoveAccount(account)
 	return nil
 }
+
+// AcquireTokenByUsernamePassword acquires a security token from the authority, via Username/Password Authentication.
+// NOTE: this flow is NOT recommended.
```

Comment:
I would detail this statement as

"The ROPC flow is not recommended, because it will block conditional access and multi-factor authentication, as these protection measures involve user interaction through a browser.

 It can still be used in special cases, such as for testing purposes and in scenarios where the resource does not allow application tokens - client_credentials."